source/2019-10-25-the-ember-times-issue-121.md

```diff
@@ -65,13 +64,11 @@ From the recording, you will also learn how to:
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [Make Your Ember App Shine with Glimmer Components ✨](https://mfeckie.dev/glimmer-components-in-ember/)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+The release date of Ember's first edition - [Ember Octane](https://blog.emberjs.com/2019/08/15/octane-release-plan.html) - is approaching and with it a new way to componentize your app: [Glimmer Components](https://emberjs.github.io/rfcs/0416-glimmer-components.html) provide a simple API to create reusable building blocks for your Ember application. Packed with lots of new features, including argument immutability, [tracked properties](https://emberjs.github.io/rfcs/0410-tracked-properties.html) and Outer HTML semantics, **Glimmer Components** will make it easier for you to manage component state and to deliver high-performant UIs.
```

Comment:
I think you need to add a `.` after the first mention of Glimmer Components and then start the sentence after that with They provide. Now the sentence got a bit confusing.

Reply:
That sounds great, I updated that accordingly :sparkles: